app.rb

```diff
@@ -19,6 +19,12 @@ class ChitterApp < Sinatra::Base
 
   post "/display_peep" do
     session[:peep] = params[:peep]
+    redirect "/peep_page"
+  end
+  get "/peep_page" do
+    $chitter = Chitter.new
+    @fullPeep = "#{$chitter.username} peeped #{$chitter.peep} at #{$chitter.time}"
```

Comment:
One good reason to follow language-specific spelling conventions is eliminating bugs: `@fullPeep` in your app.rb, but `@full_peep` in your peep_display.erb